### PR TITLE
Isolates simple sample

### PR DESCRIPTION
**Pull request by ianffcs**

### CODE REVIEW

**Comment by ianffcs** [comment]
@cgrand / @dupuchba I think the example spawns only one Isolate everytime the button is Pressed, I was thinking on making  a more advanced example using buttons to control the Isolates but I think this is the simple one. Do you folks think that it makes sense to build it here or to add to a new example?

File: samples/isolates/src/sample/isolates.cljd [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+           mike-send-port
+           [message-box
+            type
+            (.-sendPort ^di/ReceivePort response-receive-port)])))
+
+
+(defn isolate-test-ui
+  []
+  (f/widget
+    :let [mike-response-receive-port (di/ReceivePort.)]
```
Comment (dupuchba) [inline samples/isolates/src/sample/isolates.cljd:40]: mike or make ^^?

Comment (dupuchba) [reply samples/isolates/src/sample/isolates.cljd:40]: no need for the `.` for constructors in cljd

**Review by dupuchba — COMMENTED** [review]
Made some comments but I realise I have to test it locally. Thanks a lot @ianffcs

File: samples/isolates/src/sample/isolates.cljd (hunk 1 above)
Comment (ianffcs) [reply samples/isolates/src/sample/isolates.cljd]: I’ve changed to my, mike was in the initial example but it diverged a little bit in the sample that I’ve made

**Comment by ianffcs** [comment]
@dupuchba, I would like to know if this sample is to much simple or not, tell me :D and let’s think on how to evolve to a more advanced one if it’ll be to another folder

**Comment by dupuchba** [comment]
@ianffcs IMO it's a good example for simple cases. Basically understanding Isolates

**Comment by cgrand** [comment]
Idea: have two buttons "busy loop on event loop" "busy loop on isolate" so that we can see the effect of using an isolate

**Pull request closed without merging**